### ISSUE TITLE
hyperparameter search

### ISSUE THREAD

**Issue by hzhz2020** [TorchSSL/TorchSSL#70]
Dear Author

Thanks for the great work.

For these benchmarking results, did you run any hyperparameter searches? What is the size of the validation set?
Looking forward to hearing from you.

Thanks,
